distributions/windows/install.py

```diff
@@ -11,11 +11,17 @@
 GET_PIP_URL = 'https://bootstrap.pypa.io/get-pip.py'
 VC_REDIST_URL = 'https://aka.ms/vs/16/release/vc_redist.x64.exe'
 
+# must be replaced in another script
+NAME = '$name'
+VERSION = '$version'
+
+assert NAME != '$' + 'name'
+assert VERSION != '$' + 'version'
+
 if len(sys.argv) < 2:
-    INSTALL_DIR = 'C:/Program Files/MindsDB'
+    INSTALL_DIR = os.path.join('C:\PROGRA~1', NAME)
```

Comment:
Is `PROGRA~1` the correct location ? looks like a weirdly named dir

Reply:
it resolves to Program Files, I'm using it because otherwise there is a weird error because of spaces in the dir name